docs-web/installation/integration-with-cis.md

```diff
@@ -0,0 +1,114 @@
+# Integration with F5 CIS
```

Comment:
I'm not sure this belongs in `docs-web/installation`.

Did you consider making this an example or separate section like https://github.com/nginxinc/kubernetes-ingress/tree/master/docs-web/app-protect ?

Reply:
it mostly concerns with IC installation
However, I think it makes sense to move it to the root folder, just for the sake of visibility